charts/iotagent-json/templates/configmap.yaml

```diff
@@ -0,0 +1,346 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "iota-json.fullname" . }}
+  labels:
+    {{- include "iota-json.labels" . | nindent 4 }}
+data:
```

Comment:
Is there a way to configure the passwords via env var, so that they can be put into a secret?

Reply:
@wistefan I did some research in this.
found this link.  https://dev.to/biplov/handling-passwords-and-secret-keys-using-environment-variables-2ei0 (it may help).

Reply:
It would only be possible if the iot-agent implementation supports that way of config. I checked the doc: https://fiware-iotagent-json.readthedocs.io/en/latest/installationguide/index.html#configuration